scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
     Parameters
     ----------
     cost_matrix : 2d-array, square, non-negative
-        A square adjacency matrix. In this implementation, :math:`A` =
-        `cost-matrix` in the objective function above.
+        The square adjacency matrix :math:`A` in the objective function above.
 
     dist_matrix : 2d-array, square, non-negative
-        A square adjacency matrix.  In this implementation, :math:`B` =
-        `dist-matrix` in the objective function above.
-
-    partial_match : 2d-array, optional, (default = None)
-        Allows the user apply a seed, fixing part of the matching between
```

Comment:
Used "seed" throughout, whereas we decided to name the variable `partial_match`.